include/xtensor/xfunction.hpp

```diff
@@ -873,9 +874,8 @@ namespace xt
     template <class align, class simd, std::size_t... I>
     inline auto xfunction_base<F, R, CT...>::load_simd_impl(std::index_sequence<I...>, size_type i) const -> simd
```

Comment:
Not sure why `load_simd_impl` does not have the same return type as `load_simd`.